### ISSUE TITLE
VS 17.6.0 breaks Extension Manager

### ISSUE THREAD

**Issue by JaredEStoddard** [loop8ack/ExtensionPackTools#75]
Extension Manager 2022

Could not load type 'Microsoft.VisualStudio.ExtensionManager.GalleryOnlineExtension' from assembly 'Microsoft.VisualStudio.ExtensionEngine, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.

**Comment by loop8ack** [comment]
Read here: #71
I haven't put an update online yet (I still need to set up AppVeyor).
If you need a quick fix, you can use the version attached there.

Edit:
AppVeyor is live: https://ci.appveyor.com/project/loop8ack/extensionpacktools
An update for the Visual Studio Marketplace is also coming.

**Comment by assarbad** [comment]
Hi there, any update of when this is coming to the marketplace? As of 17.6.5, I still get:

```
---------------------------
Microsoft Visual Studio
---------------------------
Extension Manager 2022

Could not load type 'Microsoft.VisualStudio.ExtensionManager.GalleryOnlineExtension' from assembly 'Microsoft.VisualStudio.ExtensionEngine, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
---------------------------
OK   
---------------------------
```

**Comment by loop8ack** [comment]
I can't reproduce it.
What have you done, where does it occur?
Which Extension Manager version are you using?

My new version is online, have you not installed it? Since I had to change the extension ID, I can't update the old version automatically, you have to install it manually. After that, automatic updates should work again.

https://marketplace.visualstudio.com/items?itemName=Loop8ack.ExtensionManager2022

**Comment by assarbad** [comment]
Let me try again ... I think I downloaded one of those you linked elsewhere but forgot to remove the old extension (different ID).

I'll get back to you shortly. Thanks for the response.

**Comment by assarbad** [comment]
Indeed, my bad. It works with the version you pointed out. I searched by ID this time and made sure the old one was removed, too.

Have a nice weekend.

**Comment by loop8ack** [comment]
Yes, the new ID is unattractive, but unfortunately there was no other way.
But new updates will come automatically via the new ID :)
